python/version_details/version_details.py

```diff
@@ -304,11 +304,18 @@ def destroy(self):
     @property
     def current_entity(self):
         """
-        The current Shotgun entity that is active in the widget.
+        The current Shotgun entity that has been loaded and is active in the widget.
         """
         return self._current_entity
 
     @property
+    def active_entity(self):
```

Comment:
@eliveikis I have updated and simplified the code with your suggestion above. Please review changes in https://github.com/shotgunsoftware/tk-framework-qtwidgets/pull/71